### DISCUSSION TITLE
Filter articles

### DISCUSSION THREAD

**Discussion by ssebastianoo** [dailydotdev/daily#439] [Feature Requests]
It would be nice to have the possibility to filter articles, for example set a filter that blocks articles which the title contains a specific word.
I'm tired of seeing all the cryptobros articles about web 3, they're so annoying, I would like to blacklist words like `crypto`, `web3` and `blockchain`

**Comment by nimrodkra** [comment]
Hi @ssebastianoo we have a solution for that. Check out this doc to learn how to block tags and sources: https://docs.daily.dev/docs/settingyourfeed/blocking-tags-sources

**Reply by ssebastianoo** [reply]
thanks!
